opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx

```diff
@@ -559,7 +559,10 @@ const StixCoreRelationshipCreationFromEntity: FunctionComponent<StixCoreRelation
                   availableEntityTypes={virtualEntityTypes}
                   handleToggleSelectAll="no"
                   entityTypes={virtualEntityTypes}
-                  additionalFilterKeys={{ filterKeys: ['entity_type'], restrictedFiltersConfig: { filterRemoving: ['entity_type'] } }}
+                  additionalFilterKeys={{
+                    filterKeys: ['entity_type'],
+                    restrictedFiltersConfig: { filterRemoving: ['entity_type'], localModeSwitching: ['entity_type'] } }
```

Comment:
this is more a general remark : I'm not fond of the names/props used in these parts of the code.

```
additionalFilterKeys={{
  filterKeys: ['entity_type'],
  restrictions: {
    preventRemoveFor: ['entity_type'],
    preventLocalModeSwithcingFor: ['entity_type'],
  }
}}
```

Or something equivalent would be self-explanatory in my opinion.

As for the original bugfix itself, it's ok :)

Reply:
fixed